landoapi/hgexports.py

```diff
@@ -71,7 +71,7 @@ def _no_line_breaks(break_string: str) -> str:
     return "".join(break_string.strip().splitlines())
 
 
-class PatchHelper(object):
+class HgPatchHelper(object):
```

Comment:
There is repeated identical code between this class and `GitPatchHelper` (e.g., `_is_diff_line`, `header`, `write`, etc.). Common code should be moved to an abstract `Helper` and that class should be extended by `GitPatchHelper` and `HgPatchHelper`.